### DISCUSSION TITLE
New release soon

### DISCUSSION THREAD

**Discussion by ajpaulingalls** [react-native-tvos/react-native-tvos#459] [General Discussions]
I saw the version was recently updated on the 0.69.6 branch to 0.69.6-3.  I'd really like to use the new auto focus feature without having to checkout and build the branch.  Any chance this could get released soon?

**Comment by douglowder** [comment]
Yes it will get released soon, hopefully end of week. We are currently testing a release candidate build.

**Reply by ajpaulingalls** [reply]
Sweet!  Thanks!
